src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py

```diff
@@ -163,31 +163,31 @@
                 "Required if args['monthly_alpha'] is false.  Is the "
                 "proportion of upslope annual available local recharge that "
                 "is available in month m."),
-            "name": "alpha_m Parameter"
+            "name": "alpha_m parameter"
         },
         "beta_i": {
             "type": "number",
             "required": True,
             "about": (
                 "Is the fraction of the upgradient subsidy that is "
                 "available for downgradient evapotranspiration."),
-            "name": "beta_i Parameter"
+            "name": "beta_i parameter"
```

Comment:
Same question here about UTF-8 beta and subscript-i

Reply:
And same response, I suspect this is a deeper issue with interpreting character sets https://github.com/natcap/invest-workbench/issues/118